app/pages/lab/workflows-table.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router';
+
+const WorkflowsTable = ({
+  handleSetStatsCompletenessType,
+  handleWorkflowSettingChange,
+  handleWorkflowStatsVisibility,
+  labPath,
+  meta,
+  project,
+  workflows
+}) => {
+  return (
+    <table className="standard-table">
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Progress</th>
+          <th>Status</th>
+          <th>Completeness statistic</th>
+          <th>Statistic visibility</th>
+        </tr>
+      </thead>
+      <tbody>
+        {workflows.map((workflow) => {
+          const progressPercentage = workflow.completeness * 100;
+          const statsCompletenessType = (workflow.configuration) ? workflow.configuration.stats_completeness_type : 'retirement';
```

Comment:
If `workflow.configuration` exists, this will default `statsCompletenessType` to `undefined`, not `retirement`.